components/search-page/SearchCollectiveCard.js

```diff
@@ -19,7 +19,7 @@ import StyledCollectiveCard from './StyledCollectiveCard';
 const SearchCollectiveCard = ({ collective, ...props }) => {
   return (
     <StyledCollectiveCard collective={collective} position="relative" {...props} data-cy="collective-card">
-      <Container p={3}>
+      <Container p={3} pt={2}>
```

Comment:
We don't want to make changes to the general design of the card. The solution should be focused on reducing the size of the tags.